apps/web/src/components/layout/app-footer.tsx

```diff
@@ -0,0 +1,30 @@
+import Link from "next/link";
+
+import { Shell } from "../dashboard/shell";
+
+export function AppFooter() {
+  return (
+    <footer className="w-full">
+      <Shell className="flex items-center justify-between">
+        <div className="text-muted-foreground text-xs font-light">
+          All rights reserved &copy;
+        </div>
+        <div className="text-right text-xs">
+          <Link
+            href="/legal/terms"
+            className="text-foreground underline underline-offset-4 hover:no-underline"
+          >
+            Terms
+          </Link>
+          <span className="text-muted-foreground/70 mx-1">&bull;</span>
+          <Link
+            href="/legal/privacy"
+            className="text-foreground underline underline-offset-4 hover:no-underline"
```

Comment:
I think legacy and tos don't need to be on every page when you are logged in

Reply:
Good point, a lot of dashboard don't even have a footer. Maybe we can move it only to marketing pages

Reply:
Yes 🔥